app/views/editions/secondary_nav_tabs/_edit_assignee.html.erb

```diff
@@ -0,0 +1,47 @@
+<% content_for :title_context, @resource.title %>
+<% content_for :page_title, "Assign person" %>
+<% content_for :title, "Assign person" %>
+
+<%= form_for @resource, url: update_assignee_edition_path(@resource) do %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds edition__edit_assignee">
+      <% unless @resource.assignee.nil? %>
+        <%= render "govuk_publishing_components/components/radio", {
+          name: "assignee_id",
+          items: [
+            { value: @resource.assigned_to_id, text: @resource.assignee, checked: true },
+            { value: "none" , text: "None" },
+          ],
+        } %>
+        <% if enabled_users(@resource).any? %>
+          <p class="govuk-body"> or </p>
```

Comment:
It would be better if we can use the [with or divider](https://components.publishing.service.gov.uk/component-guide/radio#with_or_divider) variation on the radio component (here) in this case so that the radio controls all remain within the same fieldset. Otherwise this could cause some confusion for assistive technology. 
In which case I think the custom CSS added for this can be removed.

Reply:
Also once the radios are combined in a single fieldset I think we should add a legend for that to explain (for screen readers) what these controls are doing. Ideally we would wrap the page heading in a `<legend>` tag but I think that would cause a problem for the alignment of the sticky panel so we can repeat the page title in a visually hidden legend as provided in the [With visually hidden heading](https://components.publishing.service.gov.uk/component-guide/radio/with_visually_hidden_heading) option.